server/base/src/main/java/org/apache/accumulo/server/conf/TableConfiguration.java

```diff
@@ -175,8 +177,9 @@ public ServiceEnvironment getServiceEnv() {
   private static CompactionDispatcher createCompactionDispatcher(AccumuloConfiguration conf,
       ServerContext context, TableId tableId) {
 
-    CompactionDispatcher newDispatcher = Property.createTableInstanceFromPropertyName(conf,
-        Property.TABLE_COMPACTION_DISPATCHER, CompactionDispatcher.class, null);
+    CompactionDispatcher newDispatcher =
+        Property.createTableInstanceFromPropertyName(conf, Property.TABLE_COMPACTION_DISPATCHER,
+            CompactionDispatcher.class, new SimpleCompactionDispatcher());
 
     if (newDispatcher == null) {
```

Comment:
If we return the default implementation instead of `null` then this logging code will never be hit. 
If the goal is to preserve existing functionality, then we could return a `new SimpleCompactionDispatcher()` on line 189.